src/ui/components/auth/Auth.tsx

```diff
@@ -16,84 +16,84 @@ import React, { useEffect, useState } from "react";
-                },
-            },
-        });
+  const validateKey = async () => {
+    setIsLoading(true)
+    const response = await fetchData({
+      url: getApiUrl('/api/key/validate'),
+      method: 'POST',
+      config: {
+        headers: {
+          authorization: `Bearer ${apiKey}`,
+        },
+      },
+    })
 
-        const body = await response.json();
+    const body = await response.json()
 
-        if (response.status === 200 && body.status === "OK") {
-            localStorageHandler.setItem(StorageKeys.API_KEY, apiKey);
-            props.onSuccess();
-        } else if (response.status === UNAUTHORISED_STATUS) {
-            setApiKeyFieldError("Invalid API Key")
-        } else {
-            setApiKeyFieldError("Something went wrong");
-        }
-
-        setIsLoading(false);
+    if (response.status === 200 && body.status === 'OK') {
+      localStorageHandler.setItem(StorageKeys.API_KEY, apiKey)
+      props.onSuccess()
+    } else if (response.status === UNAUTHORISED_STATUS) {
+      setApiKeyFieldError('API key doesn’t exist')
+    } else {
+      setApiKeyFieldError('Something went wrong')
     }
 
-    const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
-        e.preventDefault();
+    setIsLoading(false)
+  }
 
-        if (apiKey !== null && apiKey !== undefined && apiKey.length > 0) {
-            validateKey();
-        } else {
-            setApiKeyFieldError("API Key field cannot be empty");
-        }
-    }
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault()
 
-    const handleApiKeyFieldChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-        const { value } = e.target;
-        setApiKey(value);
-        setApiKeyFieldError("");
+    if (apiKey !== null && apiKey !== undefined && apiKey.length > 0) {
+      validateKey()
+    } else {
+      setApiKeyFieldError('API Key field cannot be empty')
     }
+  }
+
+  const handleApiKeyFieldChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = e.target
+    setApiKey(value)
+    setApiKeyFieldError('')
+  }
 
-    return (
-        <div className="page-container">
-            <div className="api-key-form-container">
-                <h1 className="api-key-form-title text-title">Enter your API Key</h1>
-                <form className="api-key-form" onSubmit={handleSubmit}>
-                    <InputField
-                        handleChange={handleApiKeyFieldChange}
-                        name="apiKey"
-                        type="text"
-                        error={apiKeyFieldError}
-                        label="API Key"
-                        value={apiKey}
-                        placeholder="Your API Key"
-                    />
+  return (
+    <div className='page-container'>
+      <div className='block-container'>
+        <img className='title-image' src={getImageUrl('star_sparkle_stars_sparkles_icon.svg')} alt='Auth Page' />
```

Comment:
Lets rename this image, also when using images from zeplin please rename them

Reply:
its renamed